internal/worker/client_test.go

```diff
@@ -134,5 +134,5 @@ func TestProxy(t *testing.T) {
 	// - request job (succeeds)
 	// - upload artifact
 	// - cancel
-	require.Equal(t, 5, proxy.calls)
+	require.Equal(t, 6, proxy.calls)
```

Comment:
nitpick: you should update the comment about this line (fix the number of calls and add the new call).